README.md

```diff
@@ -59,30 +51,41 @@ In order to deploy your own `everware` instance, you have to:
 ```
  - [Create a Github OAuth application](https://github.com/settings/applications/new) with URL `http://localhost:8000/` and callback URL `http://localhost:8000/hub/oauth_callback`
  - Enter you OAuth information into `env.sh` and `source env.sh`
- - Install everware
+ - Install `everware`
 ```
-    pip3 install -e .
+    pip3 install -r requirements.txt -e .
 ```
- - If you are not using `boot2docker`, but run directly on Linux, edit `jupyterhub_config.py` as indicated in the file.
- - Run
+ - If you are on **Mac OS** make sure the VM in which docker runs is
+   started (`docker-machine start <vm-name>`) and your environment is
+   setup properly (`docker-machine env <vm-name>`). Then start `everware`
+   with
 ```
-    jupyterhub
+    everware-server -f etc/local_dockermachine_config.py --debug --no-ssl
 ```
+ - If you are on **Linux** make sure `dockerd` is running and your
+   environment contains the required information (`DOCKER_HOST` is set, etc).
+   Then start `everware` with
+```
+    everware-server -f etc/local_config.py --debug --no-ssl
+```
+
 
 ## Development
 
-To run our tests you will need [selenium] and [phantomjs]. On OS X you can install them with:
+Follow the instructions for deploying your own everware instance. In
+addition to run our tests you will need [selenium] and [firefox]. On
+OS X you can install selenium with:
 
 ```
 brew install selenium-server-standalone
-brew install phantomjs
```

Comment:
Why did you remove it? It's much more convenient to use it instead of firefox as for me